dbms/src/Interpreters/JoinPartition.h

```diff
@@ -172,7 +172,7 @@ class JoinPartition
         const JoinBuildInfo & join_build_info,
         ProbeProcessInfo & probe_process_info,
         MutableColumnPtr & record_mapped_entry_column);
-    template <ASTTableJoin::Kind KIND, ASTTableJoin::Strictness STRICTNESS, typename Maps>
+    template <ASTTableJoin::Kind KIND, ASTTableJoin::Strictness STRICTNESS, typename Maps, bool row_flagged_map>
```

Comment:
It seems we can get `row_flagged_map` from `Maps`?

Reply:
It would be not that straight forward. Use row_flagged_map would be clearer.

Reply:
OK